messages/config.json

```diff
@@ -3,6 +3,7 @@
   "InvalidConfigValue": "Invalid config value. %s",
   "InvalidInstanceUrl": "Specify a valid Salesforce instance URL",
   "InvalidApiVersion": "Specify a valid Salesforce API version, for example, 42.0",
+  "InvalidCustomOrgMetadataTemplates": "Specify a valid Custom Org Metadata Templates repository URL or location",
```

Comment:
As I understand it, this config value will simply point to a directory that contains the templates, right?  i.e. it's NOT going to automatically clone/download a repo?  If so, I suggest this wording:

"Specify a valid directory that contains the custom project templates for working with org metadata."

Okay? @mshanemc , I'd love your opinion on the wording.  For example, is it weird say "for working with org metadata" when there's currently only one type of templates?  This is part of the "thinking aheadd when the CLI works with all clouds", but is it confusing to mention it now?

Reply:
Also, is there an equivalent environment variable?  If so, what is it? Thx!

Reply:
> this config value will simply point to a directory that contains the templates, right? i.e. it's NOT going to automatically clone/download a repo
It's not going to automatically clone/download a repo when `config:set` command is run, but you can specify a repo URL as the config value and the client (CLI / VS Code) will download the repo when creating with the custom templates.

Reply:
The env variable would be automatically applied like other env variables: `SFDX_CUSTOM_ORG_METADATA_TEMPLATES`

Reply:
Gotcha.  So now I have just a small edit to that error message:

"Specify a valid repository URL or directory for the custom org metadata templates."

Reply:
Thanks, updated the message!